doc/developers/contributing.rst

```diff
@@ -210,37 +203,25 @@ then submit a "pull request" (PR):
        $ git clone git@github.com:YourLogin/scikit-learn.git
        $ cd scikit-learn
 
-4. Install library in editable mode::
+4. Install the development dependencies::
+
+       $ pip install numpy scipy cython pytest flake8
+
+5. Install scikit-learn in editable mode::
 
        $ pip install --editable .
 
    for more details about advanced installation, see the
    :ref:`install_bleeding_edge` section.
 
-5. Create a branch to hold your development changes::
+6. Create a branch to hold your development changes::
 
        $ git checkout -b my-feature
 
    and start making changes. Always use a ``feature`` branch. It's good practice to
    never work on the ``master`` branch!
 
-   .. note::
```

Comment:
I thought this note was redundant with the instructions below about solving conflicts, but feel free to disagree (esp @adrinjalali since I think you wrote it IIRC).

Reply:
agreed